appveyor.yml

```diff
@@ -43,7 +43,8 @@ build_script:
     - cd _build
     - ps: >-
         If ($env:Configuration -Match "shared" -or $env:EXTENDED_TESTS) {
-            cmd /c "nmake 2>&1"
+            cmd /c "nmake build_all_generated 2>&1"
+            cmd /c "nmake PERL=no-perl 2>&1"
```

Comment:
Oh!  I was thinking something a bit more selective...  but ok (and yeah, gotcha, since we configure `no-makedepend` anyway...)

Reply:
Well, there is no need to be more selective, but what one can do is to split them so that if build_all_generated fails, then PERL=no-perl doesn't even start. But it wouldn't actually make big difference for troubleshooting, because PERL=no-perl is likely to fail very early.